cmd/nfd-topology-updater/main.go

```diff
@@ -143,6 +144,7 @@ func initFlags(flagset *flag.FlagSet) (*topology.Args, *resourcemonitor.Args) {
 		"Pod Resource Socket path to use.")
 	flagset.StringVar(&args.ConfigFile, "config", "/etc/kubernetes/node-feature-discovery/nfd-topology-updater.conf",
 		"Config file to use.")
+	flagset.StringVar(&resourcemonitorArgs.KubeletDirPath, "kubelet-dir-path", DefaultKubeletDirPath, "Kubelet directory path for watching state and checkpoint files")
```

Comment:
nit: I think this commit isn't buildable as `resourcemonitorArgs.KubeletDirPath` is not defined. Also, this parameter doesn't have anything to do with resourcemonitor so maybe not specify the arg there.

Reply:
My bad. Fixed